src/hub.html

```diff
@@ -453,6 +470,24 @@
                 </a-entity>
             </template>
 
+            <template id="linked-media">
+                <a-entity
+                    class="interactable"
+                    body-helper="type: dynamic; mass: 1; collisionFilterGroup: 1; collisionFilterMask: 15;"
+                    is-remote-hover-target
+                    tags="offersRemoteConstraint: true;"
+                    set-unowned-body-kinematic
+                    destroy-at-extreme-distances
+                    scalable-when-grabbed
```

Comment:
Same as above -- are you supposed to be able to grab and scale this object?

Reply:
no, you shouldn't be able to at this time (tho eventually I'd like to let you do that, and then when you let go, it returns to its normal position)

Reply:
For now you can remove `scalable-when-grabbed` (since this object won't be grabbed). That'll be a nice UX for the future though.